### PR TITLE
RPRBLND-1896: Case scene.world == None isn't handled by plugin.

### PR DESCRIPTION
**Pull request by DagerD**
### PURPOSE
There is errors in console during rendering and UI operations if we delete world light from scene.

### EFFECT OF CHANGE
No more errors if world light doesn't exist.

### TECHNICAL STEPS
Added checks for world light everywhere it is needed

### CODE REVIEW

File: src/rprblender/ui/world.py [hunk 1]
```diff
@@ -24,13 +24,19 @@ class RPR_WORLD_PT_environment(RPR_Panel):
     bl_context = 'world'
 
     def draw_header(self, context):
```
Comment (bnagirniak) [inline src/rprblender/ui/world.py:26]: override poll()

File: src/rprblender/ui/world.py [hunk 2]
```diff
@@ -163,9 +182,15 @@ class RPR_WORLD_PT_sun_sky(RPR_Panel):
 
     @classmethod
     def poll(cls, context):
+        if context.scene.world is None:
+            return super().poll(context)
```
Comment (bnagirniak) [inline src/rprblender/ui/world.py:186]: return False

File: src/rprblender/engine/render_engine.py [hunk 3]
```diff
@@ -672,12 +672,13 @@ def sync(self, depsgraph):
             self.camera_data.export(rpr_camera)
 
         # Environment is synced once per frame
-        if scene.world.is_evaluated:  # for some reason World data can came in unevaluated
-            world_data = scene.world
-        else:
-            world_data = scene.world.evaluated_get(depsgraph)
-        world_settings = world.sync(self.rpr_context, world_data)
-        self.world_backplate = world_settings.backplate
+        if scene.world is not None:
```
Comment (bnagirniak) [inline src/rprblender/engine/render_engine.py:675]: simplify to: `if scene.world:`

**Review by bnagirniak — CHANGES_REQUESTED** [review]

**Comment by DagerD** [comment]
Close this PR due to wrong target branch. Will create new.

**Pull request closed without merging**